apps/blackwhite/style.postcss

```diff
@@ -44,15 +44,15 @@ main {
 	padding: .3em .5em;
 	border-radius: 1px;
 
-	@nest [data-order="l"] & {
+	[data-order="l"] & {
 		order: var(--l);
 	}
 
-	@nest [data-order="c"] & {
+	[data-order="c"] & {
 		order: var(--c);
 	}
 
-	@nest [data-order="h"] & {
+	[data-order="h"] & {
```

Comment:
These were [causing deprecation warnings](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-nesting#%EF%B8%8F-nest-has-been-removed-from-the-specification).